config/game_options.txt

```diff
@@ -468,7 +468,7 @@ ROUNDSTART_RACES ipc
 ROUNDSTART_RACES ethereal
 ROUNDSTART_RACES apid
 ROUNDSTART_RACES synthetic
-#ROUNDSTART_RACES jelly
+ROUNDSTART_RACES jelly
```

Comment:
config changes like these tend not to actually make it onto the live server and need to be done per hostbox

Reply:
Acknowledged that in the thing but I'll change it anyhow, this is pretty much completely inconsequential nomatter which way it goes

Reply:
there it goes